### ISSUE TITLE
Release tarball doesn't contain Uninstall.cmake.in

### ISSUE THREAD

**Issue by refi64** [libevent/libevent#873]
This causes the following build error:

```
CMake Error: File /run/build/libevent/cmake/Uninstall.cmake.in does not exist.
CMake Error at CMakeLists.txt:1498 (configure_file):
  configure_file Problem configuring file
```

**Comment by ygj6** [comment]
There is a bug in `release-2.1.11-stable`. Please build it via `autotconf/automake` instead of `cmake`, the commands are:
```
./autogen.sh
mkdir build
cd build
../configure --prefix=/your/install/path
make && make install
```
If you really want to use `cmake`, you should download the [Source code](https://github.com/libevent/libevent/archive/release-2.1.11-stable.tar.gz) or  clone the git repository  instead of `libevent-2.1.11-stable.tar.gz`.

It already fixed in #863

**Comment by refi64** [comment]
Ah sorry for the noise, can't believe I missed that previous bug report...
